### PR TITLE
new packages: sfcgal and pysfcgal

### PR DESCRIPTION
**Pull request by lbartoletti**
Hello!

I would like to propose two new packages into MSYS2: SFCGAL and pysfcgal.

Some words about SFCGAL:

 > SFCGAL is a C++ wrapper library around CGAL with the aim of supporting ISO 19107:2013 and OGC Simple Features Access 1.2 for 3D operations.
 SFCGAL provides standard compliant geometry types and operations, that can be accessed from its C or C++ APIs. PostGIS uses the C API, to expose some SFCGAL's functions in spatial databases (cf. PostGIS manual). 

http://www.sfcgal.org/

pysfcgal is the python binding of SFCGAL using cffi.

As it's my first PR here, a tiny description about me:

I'm the main maintainer/developer of SFCGAL and pysfcgal, and I'm also a (FreeBSD) package maintainer. I intend to make changes with each new version and if there are bugs.

### CODE REVIEW

**Comment by Biswa96** [comment]
Thank you for your contribution.

Though this is not a requirement, ninja is preferred to build packages with cmake. Is it possible to use ninja to build sfcgal?

File: mingw-w64-python-pysfcgal/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+pkgver=0.1.0
+pkgrel=1
+pkgdesc='Python SFCGAL binding (mingw-w64)'
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+license=('MIT')
+url='https://gitlab.com/Oslandia/pysfcgal'
+depends=(
+    ${MINGW_PACKAGE_PREFIX}-python
+    ${MINGW_PACKAGE_PREFIX}-sfcgal
```
Comment (naveen521kk) [inline mingw-w64-python-pysfcgal/PKGBUILD:15]: As the [check](https://github.com/msys2/MINGW-packages/runs/4458689755?check_suite_focus=true#step:6:70) says
> pysfcgal 0.1 requires cffi, which is not installed.

Add, `${MINGW_PACKAGE_PREFIX}-python-cffi` here.

Comment (lbartoletti) [reply mingw-w64-python-pysfcgal/PKGBUILD:15]: Done. thanks

**Comment by lbartoletti** [comment]
> Though this is not a requirement, ninja is preferred to build packages with cmake. Is it possible to use ninja to build sfcgal?

Yes, I pushed a new commit with ninja.

**Comment by lbartoletti** [comment]
Do I need to alter a few more things for it to merge?

**Comment by naveen521kk** [comment]
Build fails for some reason, 
```py
  creating build/temp.mingw_x86_64-3.9
  error: [WinError 2] The system cannot find the file specified
  ==> ERROR: A failure occurred in build().
```
Is it a missing dep on `${MINGW_PACKAGE_PREFIX}-cc`?

**Comment by lbartoletti** [comment]
> Is it a missing dep on ${MINGW_PACKAGE_PREFIX}-cc?

@naveen521kk Yes, thanks!

File: mingw-w64-sfcgal/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+_realname=sfcgal
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=1.4.1
+pkgrel=1
+pkgdesc="Wrapper around CGAL that intents to implement 2D and 3D operations on OGC standards models (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+url="https://www.sfcgal.org/"
+license=("LGPL, GPL")
```
Comment (naveen521kk) [inline mingw-w64-sfcgal/PKGBUILD:12]: Use spdx prefixed license https://www.msys2.org/dev/package-licensing/#the-license-array-field

**Comment by lbartoletti** [comment]
Thanks.

**Pull request merged**